sos/plugins/npm.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2014 Red Hat, Inc., Tomas Tomecek <ttomecek@redhat>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, \
+    SuSEPlugin
+
+
+class Npm(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, SuSEPlugin):
+    """
+    Get info about available npm modules
+    """
+
+    requires_root = False
```

Comment:
I discovered that attribute on `Plugin` class by accident. And actually bumped into this when I tried to invoke `sosreport` as an unprivileged user and was surprised it can't find any plugins.